bin/decorate-silo-mode-tests

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+import ast
```

Comment:
I'm thinking that once we have the models-manifest, we can take that metadata and automatically tag all the tests associated with a model with the appropriate decorator, and set it to stable mode when ready.

Reply:
So this now works.  There's a natural workflow:

First, decide which silo mode the model should work in, and make sure all related tests are decorated consistently
`bin/decorate-silo-mode-tests UserIP --silo-mode control`

Second, `SILO_MODE_UNSTABLE_TESTS=1 bin/run-model-tests.py UserIP` will execute all tests in the manifest file associated with the UserIP model, and run them with silo tests activated.

Thirdly, make changes to get these tests passing!  For tests that are still failing in more fundamental ways (or are multi model tests), we can skip them for now.  Once the vast majority of model tests are passing, decorate them as stable: `bin/decorate-silo-mode-tests UserIP --silo-mode control --set-stable`

Fourthly, make PR.  Now all the UserIP model tests would run in monolith and silo mode by default!  Check the CI to ensure things pass and test markings make sense.  Remove any markings from tests that are unstable or still in need of fixing.

Fifth, merge and repeat process until all test models and their tests pass in CI.